### ISSUE TITLE
SoftDeletesMixin doesn't work as expected when trying to change the column name. 

### ISSUE THREAD

**Issue by JRubics** [MasoniteFramework/orm#613]
SoftDeletesMixin doesn't work as expected when trying to change the column name. 

When we try to add `__deleted_at__ = 'archived_at'` to the Model class, it changes the value of __deleted_at__ field on the class but it is not used later when creating a query. I think the reason is this in masoniteorm.models.Model, method boot, [this](https://github.com/MasoniteFramework/orm/blob/99f32b18b1c5e227edb156ecd2d6205bde63e73e/src/masoniteorm/models/Model.py#L280) line:

```
if class_name.endswith("Mixin"):
       getattr(base_class(), "boot_" + class_name)(self.builder)
```
because it doesn't use previously changed mixin, but creates a new one. 

I think it can be solved like this:
```
if class_name.endswith("Mixin"):
       getattr(self, "boot_" + class_name)(self.builder)
```
but i am not sure if it will break some other mixin.

Can you please tell me if this change is safe to add, and I will create a PR if it is. If not, do you have some other solution to my problem?

Desktop:

    OS: Linux
    Version Ubuntu 21.10

What database are you using?

    Type: [MySQL]
    Version [14.14 Distrib 5.7.30, for Linux (x86_64) using EditLine wrapper]
    Masonite ORM [2.4.2]

**Comment by Marlysson** [comment]
@JRubics Do you inherit your model class from SoftDeletesMixin like this:

```python
class User(Model, SoftDeletesMixin):
```

Maybe the error is here:

https://github.com/MasoniteFramework/orm/blob/99f32b18b1c5e227edb156ecd2d6205bde63e73e/src/masoniteorm/scopes/SoftDeletesMixin.py#L7

With __delete_at__ column defined hardcoded instead get from model located in the builder.

You can create a PR with some tests and run them safely that we will analyze and just merged with everything ok.

**Comment by JRubics** [comment]
@Marlysson I did and exactly this is the problem because boot method in Model class creates a new instance of the SoftDeletesMixin that doesn't contain my change but only hardcoded value. Okay, I'll try tomorrow :)

**Comment by josephmancuso** [comment]
This is fixed in latest 2.5.0 version

**Comment by JRubics** [comment]
Thank you! :tada:
